src/endpoints/Auth.php

```diff
@@ -36,7 +37,7 @@ public function __invoke(Application $app)
         $app->post('/logout/{user}', [$this, 'logoutFromAll']);
         $app->post('/logout/{user}/{id}', [$this, 'logoutFromOne']);
         $app->post('/password/request', [$this, 'forgotPassword']);
-        $app->get('/password/reset/{token}', [$this, 'resetPassword']);
+        $app->post('/password/reset', [$this, 'resetPassword']);
```

Comment:
Passwords are now being passed in the body instead of being auto-generated.

The previous method was done that way because the API didn't know where the app was, so by utilizing GET requests, we could circumvent that. The UX now is way nicer